src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -17,11 +17,14 @@ namespace Microsoft.ML
     public static class TextCatalog
     {
         /// <summary>
-        /// Transform a text column into featurized float array that represents counts of ngrams and char-grams.
+        /// Create a <see cref="TextFeaturizingEstimator"/>, which transforms a text column into featurized float array that represents normalized counts of ngrams and char-grams.
```

Comment:
>float array [](start = 108, length = 11)

I think it would be better to avoid float array?
Maybe `numeric vector column`? Here and below

Reply:
I put System.Single, since numeric implies int, uint etc. 

---
In reply to: [277057968](https://github.com/dotnet/machinelearning/pull/3438#discussion_r277057968) [](ancestors = 277057968)